### PR TITLE
New guide for updating DWN records

### PR DESCRIPTION
**Pull request by bobbilee19**

### CODE REVIEW

File: site/docs/web5/build/decentralized-web-nodes/update-to-dwn.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  message: {
+    recordID: record.id,
+    filter: {
+      dataFormat: 'text/plain',
+    }
+  }
+});
+
+// Update the record
+const updateResult = await record.update({ data: "Hello, I'm updated!" });
```
Comment (angiejones) [inline site/docs/web5/build/decentralized-web-nodes/update-to-dwn.mdx:21]: This is using an already existing (unseen) `record` object, so the query is not necessary. Couple of options here:

1. remove the query (lines 11-18)
2. change the query to look for the string record id to obtain the record in object form, then use the query results to get the first object and update it:

```js
// Query for the particular record
const {records} = await web5.dwn.records.query({
  message: {
    recordID: 'bafyreiaz5oycqbrnmmpvffxqyoqxvx6bcnqueprmt2qnvzcurpc52r5uyy',
    filter: {
      dataFormat: 'text/plain',
    }
  }
});
// Update the record
const status = await records[0].update({ data: "Hello, I'm updated!" });
```

**Review by angiejones — CHANGES_REQUESTED** [review]

File: site/docs/web5/build/decentralized-web-nodes/update-to-dwn.mdx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Update DWN Records
+
+Before writing to your Decentralized Web Node (DWN), be sure to check out our [Quickstart guide](../../quickstart) to ensure you’ve properly imported the Web5 SDK. Additionally, make sure you’ve set up a DID before attempting to update data in a DWN.
+
+## Updating an exisiting record in a DWN
+
+The following snippet allows you to query and update a particular record in your `Web5` instance’s DWN:
+
+```jsx
```
Comment (angiejones) [inline site/docs/web5/build/decentralized-web-nodes/update-to-dwn.mdx:9]: just put **js** not jsx

File: site/docs/web5/build/decentralized-web-nodes/update-to-dwn.mdx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+// Query for the particular record
+const { records } = await web5.dwn.records.query({
+  message: {
+    recordID: 'bafyreiaz5oycqbrnmmpvffxqyoqxvx6bcnqueprmt2qnvzcurpc52r5uyy',
+    filter: {
+      dataFormat: 'text/plain',
+    }
+  }
+});
+// Update the record
```
Comment (angiejones) [inline site/docs/web5/build/decentralized-web-nodes/update-to-dwn.mdx:19]: add line break for readability

**Review by angiejones — APPROVED** [review]
left a couple of nit comments. take em or leave em

File: site/docs/web5/build/decentralized-web-nodes/update-to-dwn.mdx (hunk 2 above)
Comment (bobbilee19) [reply site/docs/web5/build/decentralized-web-nodes/update-to-dwn.mdx:9]: oh this is from write file

**Review by angiejones — APPROVED** [review]

**Pull request merged**